tools/PI/DevHome.PI/ViewModels/BarWindowViewModel.cs

```diff
@@ -51,6 +51,9 @@ public partial class BarWindowViewModel : ObservableObject
     [ObservableProperty]
     private Visibility _appBarVisibility = Visibility.Visible;
 
+    [ObservableProperty]
+    private string? _applicationName;
```

Comment:
NIT: remove `?` and assign `string.empty` in the constructor.

Reply:
Ok. Seems to fit the pattern better for the rest of the file. I'm guessing a tooltip of null is different than a tooltip of "", but in theory the user won't have access to the tooltip of an app if we're not attached to an app.